iBudget/static/src/components/Icon_getter.vue

```diff
@@ -1,10 +1,21 @@
 <template>
     <div id="icon_getter">
+            <p>Please choose the icon for your new {{tabName}} from the icons below or upload your own picture:</p>
             <input type = image
             v-for="icon in icons" :src="icon.path"
             v-on:click="get_name(icon.name)"
-            class="icon" alt="icon"></input>
+            class="icon" alt="icon" v-if="! upload"></input>
+            <br>
+            <br>
```

Comment:
Maybe it will be better to do this with css styles like margin or padding?
or it is necessary to use `<br>`?

Reply:
will be done!